Branch-SDK/src/io/branch/indexing/ContentDiscoveryManifest.java

```diff
@@ -35,40 +35,43 @@
     private boolean isCDEnabled_ = false;
     /* Json Array for the content path object and the filtered views for this application */
     private JSONArray contentPaths_;
-
+    
     public static final String MANIFEST_VERSION_KEY = "mv";
     public static final String PACKAGE_NAME_KEY = "pn";
-    public static final String HASH_MODE_KEY = "h";
+    static final String HASH_MODE_KEY = "h";
     private static final String MANIFEST_KEY = "m";
     private static final String PATH_KEY = "p";
     private static final String FILTERED_KEYS = "ck";
     private static final String MAX_TEXT_LEN_KEY = "mtl";
     private static final String MAX_VIEW_HISTORY_LENGTH = "mhl";
     private static final String MAX_PACKET_SIZE_KEY = "mps";
     public static final String CONTENT_DISCOVER_KEY = "cd";
-    private static final String DISCOVERY_REPEAT_TIME = "drt";
-
-
+    private static final String DISCOVERY_REPEAT_INTERVAL = "dri";
+    private static final String MAX_DISCOVERY_REPEAT = "mdr";
+    
+    static final int DEF_MAX_DISCOVERY_REPEAT = 15; // Default Maximum number for discovery repeat
```

Comment:
Oh.. it's defined in the other class as `DRT_MINIMUM_THRESHOLD`. Gross. Can you keep your constants defined consistently?